src/input_utils.py

```diff
@@ -0,0 +1,87 @@
+import attr
+import tensorflow as tf
+import tensorflow_text as tf_text
+
+from typing import List
+
+from google_research.etcmodel.models.modeling import EtcConfig 
+
+
+@attr.s
+class PretrainInputConfig(object):
+  """Config options for pretraining model input."""
+
+  image_size = attr.ib(default=224)
```

Comment:
Please add some docstring that specidies that "image_size" and "patch_size" define images of "image_size * imate_size" pixels and patches of "patch_size * patch_size" pixels.